packages/core/admin/admin/src/content-manager/components/BlocksEditor/Components.js

```diff
@@ -0,0 +1,95 @@
+import { Typography } from '@strapi/design-system';
+import styled, { css } from 'styled-components';
+
+export const H1 = styled(Typography).attrs({ as: 'h1' })`
+  font-size: ${42 / 16}rem;
```

Comment:
Genuine question since I don't know how this works. How do we know if something needs `pxToRem` vs settings the rems directly?

Reply:
otherwise it will be difficult to check/read in px which we are used to, I guess.